Configuration/webapp/app/services/session.ts

```diff
@@ -0,0 +1,47 @@
+/**
+ * @license
+ * Copyright 2019-2020 CERN and copyright holders of ALICE O2.
+ * See http://alice-o2.web.cern.ch/copyright for details of the copyright holders.
+ * All rights not expressly granted are reserved.
+ *
+ * This software is distributed under the terms of the GNU General Public
+ * License v3 (GPL Version 3), copied verbatim in the file "COPYING".
+ *
+ * In applying this license CERN does not waive the privileges and immunities
+ * granted to it by virtue of its status as an Intergovernmental Organization
+ * or submit itself to any jurisdiction.
+ */
+
+const sessionData: Record<string, string> = {};
+
+/**
+ * Function for fetching session data from the Control server
+ */
+export async function fetchSessionData() {
+  // only to get the data from server redirect
+  // (line 264, commit 3ba4600 of github.com/AliceO2Group/WebUi/blob/dev/Framework/Backend/http/server.js)
+  // this should be replaced with endpoint designed for authentication only
+  const response = await fetch('http://localhost:8080/control');
```

Comment:
You should not be using the `host` and `port` in the request as these can change depending on environment. The browser native method `fetch` is smart enough to figure those by itself.
It will work the same with:
```js
const response = await fetch('control');
```